### PR TITLE
add layout Main

### PR DESCRIPTION
**Pull request by Teiko19**
added layout main and child components
image

vercel build: https://tom-app-arlu-2dgdl0ubj-teiko19.vercel.app/
youtrack: https://emibel.youtrack.cloud/issue/RAM-1

### CODE REVIEW

**Comment by Teiko19** [comment]
youtrack: https://emibel.youtrack.cloud/issue/RAM-1

File: pages/index.tsx [hunk 1]
```diff
@@ -12,10 +13,10 @@ export default function Home() {
       <Head>
         <title>TomApp</title>
       </Head>
-      <NavBar />
-      <main>
+      <Main className="mx-3 mt-3">
```
Comment (emibel) [inline pages/index.tsx:16]: remove class names we don't need to pass valeus right now

File: src/components/main/main.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+
+export const Main = ({ children, className }) => {
+  return (
+    <div className={`mb-3 mx-0 ${className}`}>
```
Comment (emibel) [inline src/components/main/main.tsx:5]: change mx-0 to mx-3

**Review by emibel — CHANGES_REQUESTED** [review]
pls fix comments

**Pull request merged**